Dockerfiles/Dockerfile-ubuntu-18.04

```diff
@@ -4,38 +4,28 @@ FROM ubuntu:18.04
 RUN \
   apt-get update && \
   apt-get -y upgrade && \
-  apt-get install -y apt-utils autoconf bison build-essential byobu curl default-jre emacs \
+  apt-get install -y apt-utils autoconf bison build-essential byobu curl default-jre dialog emacs \
     fftw3-dev flex gdb g++-8 gcc-8 gfortran git git-lfs htop libblas-dev libcurl4-gnutls-dev \
     libpugixml-dev libcfitsio-dev libgtest-dev libhdf5-dev liblapack-dev libncurses-dev \
     libreadline-dev libssl-dev libstarlink-ast-dev libtbb-dev libtool \
     libxml2-dev libzstd-dev libgsl-dev man pkg-config python-pip python3-pip \
-    software-properties-common unzip vim wcslib-dev wget cmake uuid-dev && \
+    software-properties-common unzip vim wcslib-dev wget uuid-dev && \
   update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 80 --slave /usr/bin/g++ g++ /usr/bin/g++-8 --slave /usr/bin/gcov gcov /usr/bin/gcov-8
 
-# Get carta dependencies
-# casacore data from Kernsuite PPA
+# carta_backend unit tests require cmake >3.12 but ubuntu 18.04 provides 3.10.2
 RUN \
-  apt-add-repository -y -s ppa:kernsuite/kern-5 && \
-  apt-get update && \
-  apt-get -y install casacore-data
-
-# carta-casacore from cartavis PPA
-RUN \
-  add-apt-repository -y ppa:cartavis/carta-casacore && \
-  apt-get update && \
-  apt-get install carta-casacore 
+  cd /root && \
+  git clone https://github.com/Kitware/CMake.git && \
+  cd CMake && git checkout v3.20.0 && ./configure && \
+  make -j 2 && make install && \
+  cd /root && rm -rf CMake
 
-# zfp from cartavis ppa
+# Get carta dependencies from the cartavis-team PPA (includes casacore-data)
 RUN \
-  add-apt-repository -y ppa:cartavis/zfp && \
-  apt-get update && \
-  apt-get install zfp
-
-# grpc from webispy ppa
-RUN \
-  add-apt-repository -y ppa:webispy/grpc && \
-  apt-get update && \
-  apt-get install -y libprotobuf-dev protobuf-compiler libgrpc++-dev libgrpc-dev protobuf-compiler-grpc googletest
+  ln -snf /usr/share/zoneinfo/$CONTAINER_TIMEZONE /etc/localtime && echo $CONTAINER_TIMEZONE > /etc/timezone && \
+  add-apt-repository -y ppa:cartavis-team/carta && \
+  apt-get -y update && \
+  apt-get -y install casacore-dev googletest libgrpc++-dev libprotobuf-dev libzfp-dev protobuf-compiler-grpc 
```

Comment:
This is unnecessary. We only have to install `casacore-dev`; the other dependencies will be installed automatically. When I have finished my current changes to the carta-casacore package, this will be called `carta-casacore-dev` (so we should use that name and only merge this PR once the new packages are in place).

Reply:
Sorry, my bad -- I forgot that these were build dependencies, not runtime dependencies.